From 6e078ff2816be0af2571d1cc8626e7de4850b4db Mon Sep 17 00:00:00 2001
From: lkundrak <lkundrak>
Date: Mon, 3 Mar 2008 18:41:49 +0000
Subject: [PATCH 08/88] Blog on opengrok

---
 RCS/Makefile |   35 +++++++++++++++++++++++++++++++++--
 1 files changed, 33 insertions(+), 2 deletions(-)

diff --git a/RCS/Makefile b/RCS/Makefile
index a20e45c..3c1e01b 100644
--- a/RCS/Makefile
+++ b/RCS/Makefile
@@ -39,13 +39,44 @@ index.html: ${SOURCES} templates/index.template scripts/blogpage.awk
 	${TIDY} -qim $@ || rm -f $@
 
 ###
+### OpenGrok blog
+###
+
+SHADOW=montage -background none -geometry +5+5 -shadow
+TN=convert -resample 50%
+BLOG3+=images/opengrok-shot1.png
+BLOG3+=images/opengrok-shot2.png
+BLOG3+=images/opengrok-shot1-tn.png
+BLOG3+=images/opengrok-shot2-tn.png
+
+JUNK+=$(BLOG3)
+JUNK+=images/opengrok-shot1.xwd-tn
+JUNK+=images/opengrok-shot2.xwd-tn
+
+images/opengrok-shot1.png: images/opengrok-shot1.xwd
+	${SHADOW} $< $@
+
+images/opengrok-shot2.png: images/opengrok-shot2.xwd
+	${SHADOW} $< $@
+
+images/opengrok-shot1-tn.png: images/opengrok-shot1.xwd
+	${TN} $< $<-tn
+	${SHADOW} $<-tn $@
+
+images/opengrok-shot2-tn.png: images/opengrok-shot2.xwd
+	${TN} $< $<-tn
+	${SHADOW} $<-tn $@
+
+blog3: ${BLOG3}
+
+###
 ### Meta...
 ###
 
 build: ${TARGETS}
 
 clean:
-	rm -f ${TARGETS}
+	rm -f ${TARGETS} ${JUNK}
 
 upload: build
-	tar cf - ${TARGETS} images/mozchomp.gif |ssh ovecka.be "tar xf - -C public_html/blog"
+	tar cf - ${TARGETS} ${BLOG3} images/mozchomp.gif |ssh ovecka.be "tar xf - -C public_html/blog"
-- 
1.6.5.2

