From 7071ff69014fd83493e10aafa47df76ba3c9728c Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Wed, 29 Apr 2009 10:54:46 +0000
Subject: [PATCH 63/88] Fix title formatting

---
 entries/shortrpm.cocot |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/entries/shortrpm.cocot b/entries/RCS/shortrpm.cocot
index 1823712..0e73482 100644
--- a/entries/shortrpm.cocot
+++ b/entries/shortrpm.cocot
@@ -1,4 +1,4 @@
-= <!-- rss:title -->{{{shortrpm}}} - Short circuitting binary RPM builds<!-- /rss:title --> =
+= <!-- rss:title -->Short circuitting binary RPM builds with shortrpm<!-- /rss:title --> =
 
 <!-- rss:description -->
 Ever wonderred why can't you do a {{{rpmbuild -bb --short-circuit}}} just
-- 
1.6.5.2

