From 9b92271d53337a2185efa5ceb1ecbfc052023ea4 Mon Sep 17 00:00:00 2001
From: lkundrak <lkundrak>
Date: Tue, 2 Jun 2009 16:26:48 +0000
Subject: [PATCH 67/88] Bug

---
 entries/gnome-feature.cocot |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/entries/gnome-feature.cocot b/entries/RCS/gnome-feature.cocot
index d67b162..865e21f 100644
--- a/entries/gnome-feature.cocot
+++ b/entries/gnome-feature.cocot
@@ -1,5 +1,5 @@
 = <!-- rss:title -->How to get a feature into Gnome<!-- /rss:title --> =
 
 <!-- rss:description -->
-[http://bugzilla.gnome.org/show_bug.cgi?id=566546]
+http://bugzilla.gnome.org/show_bug.cgi?id=566546
 <!-- /rss:description -->
-- 
1.6.5.2

