From 27c05f0f3929494206ee393319f4d81d732ebc5e Mon Sep 17 00:00:00 2001
From: lkundrak <lkundrak>
Date: Mon, 9 Nov 2009 16:33:14 +0000
Subject: [PATCH 85/88] A typo

---
 entries/inspiron-12-opensuse.cocot |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/entries/inspiron-12-opensuse.cocot b/entries/RCS/inspiron-12-opensuse.cocot
index 980f363..92fbea7 100644
--- a/entries/inspiron-12-opensuse.cocot
+++ b/entries/inspiron-12-opensuse.cocot
@@ -39,7 +39,7 @@ http://www.blogcdn.com/www.engadget.com/media/2009/02/2-1-09-dell-mini-12-splaye
 == Software ==
 
 This is the most poorly supported machine by Linux that I've touched in ages.
-A complete sucker. Which is good, since with a little effort most each piece
+A complete sucker. Which is good, since with a little effort mostly each piece
 of its hardware is usable.
 
 === Installing ===
-- 
1.6.5.2

